### DISCUSSION TITLE
$reason or $message

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#2996] [Ideas]
when blocking webpages, it could be helpful to assert whether website X is not related to website Y etc, or whether a website is in fact an advertorial website rather than a random blog, etc.
```
||example^$doc,message=(This is a message)
```

**Comment by iam-py-test** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/1195
Thanks
